src/extensions/default/bramble/nohost/HTMLServer.js

```diff
@@ -143,17 +143,25 @@ define(function (require, exports, module) {
         if(liveDocument) {
             serve(liveDocument.getResponseData().body);
         } else {
-            fs.readFile(path, "utf8", function(err, body) {
+            fs.readFile(path, "utf8", function(err, content) {
                 if(err) {
                     return callback(err);
                 }
 
                 // Since we're not instrumenting this doc fully for some reason,
                 // at least inject the scroll manager so we can track scroll position.
-                body = body.replace(/<\/\s*head>/,
+                // Try to inject the scripts at the end of the <head> element
+                // if it is present, otherwise at the end of the <html> element.
+                var headRegex = new RegExp(/<\/\s*head>/);
+                var matchPattern = headRegex.test(content) ? headRegex : new RegExp(/<\/\s*html>/);
```

Comment:
What if there also isn't an `<html>` tag?  Should we just jam it at the end of the content anyway?  That's sort of what's happening in the instrumentation code above.  Seems like this is the bug to deal with this, since you know someone will complain about that next if we don't solve it now.